[2024-01-31 13:30:01,876] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.get_data_covid manual__2024-01-31T13:30:00.630819+00:00 [queued]>
[2024-01-31 13:30:01,882] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.get_data_covid manual__2024-01-31T13:30:00.630819+00:00 [queued]>
[2024-01-31 13:30:01,882] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 13:30:01,882] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-01-31 13:30:01,882] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 13:30:01,891] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_data_covid> on 2024-01-31 13:30:00.630819+00:00
[2024-01-31 13:30:01,895] {standard_task_runner.py:52} INFO - Started process 6179 to run task
[2024-01-31 13:30:01,898] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'get_data_covid', 'manual__2024-01-31T13:30:00.630819+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmphwbvncbs', '--error-file', '/tmp/tmph83v7r4t']
[2024-01-31 13:30:01,899] {standard_task_runner.py:80} INFO - Job 115: Subtask get_data_covid
[2024-01-31 13:30:01,941] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.get_data_covid manual__2024-01-31T13:30:00.630819+00:00 [running]> on host d99f9e65d396
[2024-01-31 13:30:01,991] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=get_data_covid
AIRFLOW_CTX_EXECUTION_DATE=2024-01-31T13:30:00.630819+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-31T13:30:00.630819+00:00
[2024-01-31 13:30:04,900] {logging_mixin.py:115} INFO - Successfully install requests
[2024-01-31 13:30:04,956] {python.py:173} INFO - Done. Returned value was: get_data_covid
[2024-01-31 13:30:04,970] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=get_data_covid, execution_date=20240131T133000, start_date=20240131T133001, end_date=20240131T133004
[2024-01-31 13:30:04,995] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-31 13:30:05,023] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
